scripts/trim-low-abund.py

```diff
@@ -100,19 +106,20 @@ def main():
     ###
```

Comment:
Instead of quitting we should let the user know that there were duplicate filenames and they were ignored.